tests/test_logger.py

```
@@ -2,272 +2,287 @@
             temp_log_file = temp_file.name
-        
+
         try:
             # Reset singleton
             Logger._instance = None
-            
+
             # Mock the log file path
-            with patch('rhoshift.logger.logger.Logger.LOG_FILE', temp_log_file):
+            with patch("rhoshift.logger.logger.Logger.LOG_FILE", temp_log_file):
                 logger = Logger()
```

Ruff (0.13.1)

Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)